server/config/analytics_test.go

```diff
@@ -0,0 +1,63 @@
+package config_test
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/kubeshop/tracetest/server/config"
+	"github.com/kubeshop/tracetest/server/config/configresource"
+	"github.com/kubeshop/tracetest/server/testmock"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAnalyticsEnabledConfig(t *testing.T) {
+	cleanEnv := func() {
+		// make sure this env is empty
+		os.Setenv("TRACETEST_DEV", "")
+	}
+
+	db := testmock.MustGetRawTestingDatabase()
+
+	setup := func(existingConfigs ...configresource.Config) config.Option {
+		repo := configresource.Repository(
+			testmock.MustCreateRandomMigratedDatabase(db),
+		)
+		for _, ec := range existingConfigs {
+			_, err := repo.Create(context.TODO(), ec)
+			if err != nil {
+				panic(err)
+			}
+		}
+
+		return config.WithConfigResource(repo)
+	}
+	t.Run("DefaultValues", func(t *testing.T) {
+		cleanEnv()
+		opt := setup()
+		cfg, _ := config.New(opt)
+
+		assert.True(t, cfg.AnalyticsEnabled())
```

Comment:
The default used to be `false`, meaning disabled. I thought it might be useful to make it enabled by default. Thoughts @kdhamric?